api/src/api/userRouter.ts

```diff
@@ -253,13 +253,19 @@ export const userRouterFactory = (
 
       const formationFormData =
         userData.businesses[userData.currentBusinessId].formationData.formationFormData;
-      const address = {
+
+      const address: FormationAddress = {
         addressLine1: formationFormData.addressLine1,
         addressLine2: formationFormData.addressLine2,
         addressCity: formationFormData.addressCity,
         addressMunicipality: formationFormData.addressMunicipality,
+        addressState: formationFormData.addressState,
+        addressCountry: formationFormData.addressCountry,
         addressZipCode: formationFormData.addressZipCode,
+        addressProvince: formationFormData.addressProvince,
+        businessLocationType: formationFormData.businessLocationType,
       };
```

Comment:
I'm confused what the point of this object is. It doesn't look like it's doing anything.

Reply:
Re-reading this I understand. We're purposefully clearing formation data but trying to persist the address.

I thought this was doing something different.